dash/testing/browser.py

```diff
@@ -447,11 +451,15 @@ def reset_log_timestamp(self):
             if entries:
                 self._last_ts = entries[-1]["timestamp"]
 
-    def visit_and_snapshot(self, resource_path, hook_id):
+    def visit_and_snapshot(self, resource_path, hook_id, assert_check=True):
         try:
-            self.driver.get(self.server_url + resource_path)
+            self.driver.get("{}/{}".format(self.server_url, resource_path))
```

Comment:
do we need to worry about someone giving us a `server_url` that ends in a `/`, or a `resource_path` that starts with `/`? Maybe add `.rstrip('/')` and `.lstrip('/')` respectively, just in case?

Reply:
agree to add `lstrip` but we should probably not touch `rstrip`, as if someone is testing a self-hosted site, we don't really have a conclude on the last '/'

Reply:
I meant `rstrip` on `self.server_url`, since we’re adding `/` immediately after it anyway... but yes, `lstrip` on `resource_path` like you did.

Reply:
🙈 was not clear after one day work in the garden.